### PR TITLE
Catch up GRETA versions of Ska packages for 2018068 Matlab release

### PR DESCRIPTION
**Pull request by jeanconn**
Catch up GRETA versions of Ska packages for 2018068 Matlab release .

### CODE REVIEW

**Comment by jeanconn** [comment]
@taldcroft Do you have any issues with adding the pyfits dependencies?  Looks like it makes the pyfits install work without error now, but I can probably work around this a different way (disable certificate check or some such) if there is a problem installing these in skare.  I'll continue to test.

I have no explanation for why this is not working now on the 64 bit VM and is working on the 32 bit VM.  Not sure if we need to understand.

File: cfg/perl_modules.cfg [hunk 1]
```diff
@@ -336,3 +340,9 @@ modules :
         # 2 tests don't seem to pass; we don't care at this time
         rm t/28exception.t
         env PYTHONHOME=${prefix_arch} make test
+  - name : Crypt::SSLeay
+    cmds :
+      build : |
+         ${prefix_arch}/bin/perl Makefile.PL --default PREFIX=${prefix}/lib/perl LIB=${prefix}/lib/perl
```
Comment (jeanconn) [inline cfg/perl_modules.cfg]: I think this needs "--lib $PREFIX_ARCH/lib".  Will confirm.

**Comment by jeanconn** [comment]
Also, this leaves GRETA with a 0.18 overall version tag.  Should that get an update?

**Comment by taldcroft** [comment]
Yes, I think the version should be brought up to date.

**Comment by jeanconn** [comment]
What should the version be on the GRETA side?  (2.18 to match HEAD side or ?)

**Comment by taldcroft** [comment]
I checked the diffs and it generally looks good to me.

For kicks, only my local copy of this branch I tried merging in master (instead of rebase).  This generates expected conflicts in 5 files but nothing that isn't straightforward to resolve.  This might be an option for a followup PR after this one (and after the next Matlab release).  That way you get `greta` back on track with all files in master with only a small set of diffs.  This puts all the history that you wanted into `greta` as well, albeit in the form of a post-facto glop of commits.

**Comment by taldcroft** [comment]
I guess one thing that confuses me is that you were making a point about the difference in Python version between `greta` and `master` (flight).  But I don't see that reflected in these diffs.  What am I missing?

**Comment by jeanconn** [comment]
How would that be reflected in these diffs?  This is a PR against greta.

**Comment by taldcroft** [comment]
Sorry, I misspoke and meant "these diffs on my terminal screen", which you obviously couldn't guess.

From your `greta_catchup_may2018` branch if I do `git diff master cfg` there is nothing that obviously  looks like a diff in Python version.  I would expect this in the Miniconda-64-* file in cfg/anaconda-64.cfg.  But maybe you were trying to say this before, that we just haven't updated Ska flight in forever, but if we did a fresh build (or an new dev Ska) we would get 2.7.12.  Is that right?

**Comment by jeanconn** [comment]
No, presently python in `master` is installed at 2.7.9 from pkgs.conda and not updated:

https://github.com/sot/skare/blob/a39ed41131f35411c9e0a497a88dca4ebc3a1b17/pkgs.conda#L37

It isn't "pinned" but I think in the single install with version specified we just end up with 2.7.9.

Seems like overall we still need to agree on ways forward in some of the ideas in https://github.com/sot/skare/pull/146 (deciding on a versions and unversions pkgs.conda, for example).

**Comment by jeanconn** [comment]
@mbaski This PR in-progress has the updates that we're considering (building / testing) for the GRETA skare update to go with the "big" Matlab tools release.  If you have concerns with any of this (especially from the Windows side which I don't see) or if there's some update that we need to look at that isn't in here, please let me know.  Thanks!

**Comment by jeanconn** [comment]
@taldcroft related to my version question above, I can't find notes describing what you intended major version "2" to signify, to know if it applies for GRETA at this PR.

**Comment by taldcroft** [comment]
I think that the "2" was the Python 2.7 version of Skare that is also Python 3 compatible (i.e. includes 3.x versions of most core packages).  So I think that 2.18 would apply here.

**Comment by jeanconn** [comment]
Thanks!

File: pkgs.manifest [hunk 2]
```diff
@@ -2,60 +2,81 @@
...
+Ska.report_ranges-0.2.1.tar.gz
+Ska.Sun-3.5.tar.gz
+Ska.Table-0.5.1.tar.gz
+Ska.tdb-3.5.1.tar.gz
+Ska.TelemArchive-0.8.1.tar.gz
+Ska.quatutil-3.3.1.tar.gz
+#
 tables-2.3.1.tar.gz
-testr-0.1.tar.gz
-Chandra.acis_esa-0.01.tar.gz
```
Comment (jeanconn) [inline pkgs.manifest]: Is this needed for anything on GRETA?  I confess I didn't pay attention to the evolution here.

**Pull request merged**